src/components/Hub/HubImagesList.tsx

```diff
@@ -12,28 +11,118 @@ import ImageCard from "./ImageCard"
 import HubFilters, { getSelectedFilters } from "./HubFilters"
 import SpinningLoader from "../Common/SpinningLoader"
 import { ExpandingSearchbar } from "../Common/ExpandingSearchbar"
-import { FilterCategory,  FilterParams } from "../../redux/hub/hub.types"
+import {
+  FilterCategory,
+  FilterParams,
+  HubImage,
+} from "../../redux/hub/hub.types"
 import styled from "@emotion/styled"
+import { Grid } from "@material-ui/core"
+import HubSortDropdown from "./HubSortDropdown"
+import { sortOptions } from "../../redux/hub/hub.constants"
 
-const SearchContainer = styled(Row)`
-  flex-direction: row-reverse;
-  padding: 1rem;
-`
+const sortHubImages = (images: HubImage[], selectedSortOption: string) => {
+  switch (selectedSortOption) {
+    case "name increasing":
```

Comment:
I think it would be good to refactor all the comparators to separate functions and test them.